### DISCUSSION TITLE
Logs are not being written inside crossplane pod

### DISCUSSION THREAD

**Discussion by sudheer757** [crossplane/crossplane#5475] [Q&A]
We have a specific requirement where I need to ship crossplane logs to splunk via fluentbit. I am currently using Aws Eks with fargate. The challenge I could see here in the crossplane pod is, logs were not getting written to any files inside pod as fluentbit can expect logs to be in the files.
Can someone helps me if crossplane writes logs to somewhere else other than stdout. If it is not possible atleast please suggest a way to pull argocd logs to the splunk via fluentbit. One big challenge here is, as we are using eks with fargate, there is no scope of getting logs from nodes.

**Comment by bobh66** [comment]
kubernetes pods typically log to stdout and rely on other mechanisms to collect the logs, for example from the files that docker maintains in `/var/log/containers` on the nodes.

**Comment by sudheer757** [comment]
@bobh66   the challenge that we have is crossplane is not writing logs to any files inside the pod and we are using eks with fargate so we won't be able to collect logs from /var/logs/containers

**Comment by bobh66** [comment]
Is this relevant: https://aws.amazon.com/blogs/containers/fluent-bit-for-amazon-eks-on-aws-fargate-is-here/

**Reply by sudheer757** [reply]
As per the above doc, so far the logging router support is enabled only for cloudwatch, elastic search and s3 only. They are planning to implement it to splunk but it's still not available.

**Comment by negz** [comment]
As @bobh66 mentions, Crossplane writes logs to stdout, not files in the pods. This is standard practice in Kubernetes.

This seems more like a request for help than a bug in Crossplane, so I'm going to convert this issue to a discussion.
